### DISCUSSION TITLE
Backup Repository Tab Missing After Configuring NAS Backup Plugin on CloudStack 4.19.1.3

### DISCUSSION THREAD

**Discussion by MarcoSpinaBZZ** [apache/cloudstack#9997] [Q&A] [answered]
Hi everyone,

I am currently trying to set up the [NAS Backup and Recovery Plugin](https://www.shapeblue.com/nas-backup-and-recovery-plugin/) in my CloudStack installation (version 4.19.1.3). I’ve followed the steps as described in the guide and made the following configurations:

Set `backup.framework.enabled` to `true`
Set `backup.framework.provider.plugin` to `nas`

However, after restarting the management server, I don’t see the Backup Repository tab in the Configurations section, which I understand should appear once the plugin is properly set up.

### Here’s what I’ve double-checked:

1. Verified that the above properties are correctly set in the `global_settings` table.
2. Restarted the CloudStack management service after making these changes.
3. Ensured that the plugin JAR is present in the `plugins/` directory (assuming this is required).

Since 4.19.1.3 is the latest version of CloudStack, I’m wondering if the issue could be related to the plugin’s compatibility with this version.

### My questions:

1. Has anyone successfully implemented the NAS Backup Plugin on CloudStack 4.19.1.3?
2. Are there additional configuration steps I might be missing to enable the Backup Repository tab?
3. If compatibility with 4.19.1.3 is uncertain, has anyone tested this plugin on older versions of CloudStack where it worked successfully? If so, which version?

Any insights or troubleshooting tips would be greatly appreciated!
Thanks in advance for your help!

**Comment by Pearl1594** [comment, marked as answer]
@MarcoSpinaBZZ  NAS backup & Recovery Plugin supported has been added in ACS 4.20 which should be released soon.

**Reply by leo79901** [reply]
cool !
